packages/plans-grid/src/plans-feature-list/style.scss

```diff
@@ -44,18 +44,21 @@ ul.plans-feature-list__item-group {
 
 	// the tick
 	& > svg path {
-		fill: var( --studio-green-40 );
-		stroke: var( --studio-green-40 );
+		fill: var( --studio-gray-100 );
+		stroke: var( --studio-gray-100 );
```

Comment:
Why are we changing the color of the tick in this case from green to gray? From the looks of it, we should only change the color of the tick from gray to orange, when the free plan is not available

Reply:
According to design specs, all checks have to be dark gray hence I updated the colors. Can you verify if this is the actual case or if I'm mistaking? :)

Reply:
It looks like this change does have an impact.

**This is how the plans grid looks like on production currently:**

In New Onboarding flow:

![Screenshot 2020-11-30 at 11 35 21](https://user-images.githubusercontent.com/1083581/100600187-3d380f00-3301-11eb-98a2-9b91c28d027b.png)

In Step by Step flow:

![Screenshot 2020-11-30 at 11 39 42](https://user-images.githubusercontent.com/1083581/100600225-4de88500-3301-11eb-8c92-dfd8f003bb16.png)


**And this is how it looks like on this PR**

In New Onboarding flow:

![Screenshot 2020-11-30 at 11 36 36](https://user-images.githubusercontent.com/1083581/100600294-65c00900-3301-11eb-8b84-f4324abc499f.png)

In Step by Step flow:

![Screenshot 2020-11-30 at 11 42 38](https://user-images.githubusercontent.com/1083581/100600308-6c4e8080-3301-11eb-88e5-d9273590bbe5.png)

---

As you can see, in production all ticks are green. Is there a way we can avoid this breaking change, since we're only supposed to change the color of the disabled plan icon to orange?